### PR TITLE
Job listing for a junior data scientist

### PR DESCRIPTION
**Pull request by vvancak**

### CODE REVIEW

File: _careers/data_scientist.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+excerpt: "As a part of our Data Science team, you will get the opportunity to design, build and test models and applications that deliver analytical and predictive content directly to users, combining commercial, analytical and technological expertise. Additionally, we contribute to open-source solutions and to present at conferences and summits. Our mission is to play a key role in helping ABSA, a huge and multinational financial corporation, to become data-driven. "
+---
+Location: {{ page.location }}
+
+### About the role
+{{ page.excerpt }}
+
+### Responsibilities
+- Work closely with the business stakeholders and technology leaders
+- Design and implementation of Data Science products in various business area
```
Comment (Zejnilovic) [inline _careers/data_scientist.md:14]: Various -> Areas
```suggestion
- Design and implementation of Data Science products in various business areas
```

File: _careers/data_scientist.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+- Research of new technologies and investigation of their usability in our systems
+
+### Must Have
+- Strong problem solving skills
```
Comment (Zejnilovic) [inline _careers/data_scientist.md:18]:
```suggestion
- Strong problem-solving skills
```

File: _careers/data_scientist.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+- Fluent written and spoken English
+- Some experience with coding, ideally python
+- Background in computer science, engineering, physics, mathematics or equivalent
+- Be eager to learn, improve and grow together with the team
+
+
+### Desirable Skills
+- Be able to creatively solve challenges, by out of the box thinking, while having an understanding of the required business needs
```
Comment (Zejnilovic) [inline _careers/data_scientist.md:26]: I think all the commas in this sentence are unnecessary but I would not stake on it.

Comment (vvancak) [reply _careers/data_scientist.md]: Ok updated + added one more requirement based on the feedback from the team

**Review by Zejnilovic — APPROVED** [review]

File: _careers/data_scientist.md (hunk 3 above)
Comment (yruslan) [reply _careers/data_scientist.md]: out of the [Jankins-Box](https://en.wikipedia.org/wiki/Box%E2%80%93Jenkins_method) thinking... 😆 
sorry, just joking

**Pull request merged**